docs/platformdev_selecting_arm_mcu.md

```diff
@@ -37,6 +37,48 @@ or one of the following being `TRUE`:
 
 For the most part, this is the bare minimum to be able to have a high confidence that QMK will be able to run on your MCU. After that, it's all up to configuration.
 
+<details>
+ <summary><b>Table of ChibiOS supported STM32 MCU with USB and/or OTG from the Dec 18,20,29, 2022 commits.</b></summary>
+
+<i>This table represents theoretical support. Not all packages will physically work due their lack of USB/OTG pins.</i>
+
+### STM32_HAS_USB TRUE
+
+|STM32F0xx  |STM32F1xx  |STM32F37x  |STM32F3xx  |STM32G0xx  |STM32G4xx  |STM32L0xx  |STM32L1xx|STM32L4xx  |STM32L5xx  |STM32WBxx  |
+|-----------|-----------|-----------|-----------|-----------|-----------|-----------|---------|-----------|-----------|-----------|
+|STM32F042x6|STM32F103x6|STM32F373xC|STM32F302x8|STM32G0B1xx|STM32G431xx|STM32L052xx|STM32L1xx|STM32L432xx|STM32L4R5xx|STM32WB55xx|
```

Comment:
This could be better spaced, as it's kind of a pain to read, here.

Reply:
the md styler **will** add a horizontal slider to it if it is displayed on a screen lacking the width clearance, see : 
![image](https://github.com/qmk/qmk_firmware/assets/98633588/9908b597-fd68-488f-a032-11383ca8ab70)

with that said yes if you're developing on a 1080p screen or at high dpi, the preview in code may not be the best but see for yourself in a IDE : 
![image](https://github.com/qmk/qmk_firmware/assets/98633588/be0f7f54-6b2e-4efb-8f0d-6489f2ae72dc)

Reply:
A friend pointed out to me that we could also not mention STM32 everytime, it would solve the concern you raised, and i'm obviously willing to make the necessary changes.

However the whole point of this table originally was to make it easier to compare price and features of each MCUs, and to do so, a simple _**double-click** → copy → paste_ into any supplier's website, to be able to start comparing features by the lower priced ones

Reply:
A friend pointed out to me that we could also not mention STM32 everytime, it would solve the concern you raised, and i'm obviously willing to make the necessary changes.

However the whole point of this table originally was to make it easier to compare price and features of each MCUs, and to do so, a simple _**double-click** → copy → paste_ into any supplier's website, to be able to start comparing features by the lower priced ones

<details>
 <summary>EDIT : said change would look like this in code : </summary>

```md
### STM32_HAS_USB TRUE

|STM32F0xx|STM32F1xx|STM32F37x|STM32F3xx|STM32G0xx|STM32G4xx|STM32L0xx|STM32L1xx|STM32L4xx|STM32L5xx|STM32WBxx|
|------|------|------|------|------|------|------|----|------|------|------|
|F042x6|F103x6|F373xC|F302x8|G0B1xx|G431xx|L052xx|L1xx|L432xx|L4R5xx|WB55xx|
|F048xx|F103xB|      |F302xC|G0C1xx|G441xx|L062xx|    |L422xx|L4R7xx|      |
|F070x6|F101xE|      |F302xE|      |G474xx|L053xx|    |L433xx|L4R9xx|      |
|F070xB|F103xE|      |F303xC|      |G484xx|L063xx|    |L443xx|L4S5xx|      |
|F072xB|F101xG|      |F303xE|      |G491xx|L072xx|    |L452xx|L4S7xx|      |
|F078xx|F103xG|      |F358xC|      |      |L073xx|    |      |L4S9xx|      |

### STM32_HAS_OTG TRUE

|STM32F1xx|STM32F4xx|STM32F7xx|STM32H7xx|STM32L4xx+|STM32L4xx|
|------|-------------|------|------|------|
|F105xC|F205xx|F722xx|H743xx|L4P5xx|L475xx|
|F107xC|F207xx|F723xx|H753xx|L4Q5xx|L476xx|
|      |F215xx|F732xx|H750xx|L4R5xx|L486xx|
|      |F217xx|F733xx|      |L4R7xx|      |
|      |F401xx|F745xx|      |L4R9xx|      |
|      |F405xx|F746xx|      |L4S5xx|      |
|      |F407xx|F756xx|      |L4S7xx|      |
|      |F411xx|F765xx|      |L4S9xx|      |
|      |F412xx|F767xx|      |      |      |
|      |F413xx|F769xx|      |      |      |
|      |F415xx|F777xx|      |      |      |
|      |F417xx|F779xx|      |      |      |
|      |F427xx|      |      |      |      |
|      |F429xx|      |      |      |      |
|      |F437xx|      |      |      |      |
|      |F439xx|      |      |      |      |
|      |F446xx|      |      |      |      |
|      |F469xx|      |      |      |      |
|      |F479xx|      |      |      |      |
```

and like this on Docsify : 
<img src="https://github.com/qmk/qmk_firmware/assets/98633588/09055d24-058f-46f3-bafd-4195b44bd92e">

as mentioned, it does not follow my initial desire but it does solve your concern. 
</details>

Reply:
> This could be better spaced, as it's kind of a pain to read, here.

Can you please explain what you meant if the previous answer i wrote aren't covering it ?